packages/dropdowns/package.json

```
@@ -23,7 +23,7 @@
   "dependencies": {
     "@floating-ui/react-dom": "^2.0.0",
     "@zendeskgarden/container-combobox": "^2.0.0",
-    "@zendeskgarden/container-menu": "^0.5.1",
+    "@zendeskgarden/container-menu": "1.0.0",
```

```suggestion
    "@zendeskgarden/container-menu": "^1.0.0",
```

All Garden-internal packages get a hat :wink: